### PR TITLE
Decapitalizing more works in breadcumbs. closes #1006

### LINKED ISSUE #1006: Global breadcrumb exceptions: words to lowercase unless they begin the text string
the
a
an
and
but
for
or
nor
to 
as

### PR DESCRIPTION
**Pull request by webinista**
Adding the following words to the "don't capitalize" list
- the
- an
- and
- but
- or
- nor
- to
- as

### CODE REVIEW

**Comment by webinista** [comment]
Assigning to you @emilyville, but reassign as needed.

File: _plugins/breadcrumbs.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class BreadcrumbsTag < Liquid::Tag
   
     def humanize(s)
       # Words that shouldn't get capitalized
-      filterwords = ['for','a']
+      filterwords = ['a', 'a', 'an', 'and', 'as', 'but', 'for', 'for', 'nor', 'or', 'the', 'to']   
       s.split("-").collect do |s|
```
Comment (emilyville) [inline _plugins/breadcrumbs.rb]: Did you mean to have "a" and "for" on here twice?

**Comment by emilyville** [comment]
(otherwise looks good to me, if that was intentional go ahead and merge)

File: _plugins/breadcrumbs.rb (hunk 1 above)
Comment (webinista) [inline _plugins/breadcrumbs.rb]: No, I did not. Thanks for catching that!

**Comment by emilyville** [comment]
Woo! Looks good.

**Pull request merged**